src/runner/browser-env/vite/browser-modules/expect.ts

```diff
@@ -0,0 +1,96 @@
+import { expect, type Expect } from "expect";
+import { ASYMMETRIC_MATCHER } from "./constants.js";
+import { BrowserEventNames, type BrowserRunExpectMatcherPayload, MockMatcherFn } from "./types.js";
+
+export const initExpect = (): Expect => {
+    const mockedMatchers = mockMatchers(window.__testplane__.expectMatchers);
+    expect.extend(mockedMatchers);
+
+    return expect;
+};
+
+function mockMatchers(matcherNames: string[]): Record<string, MockMatcherFn> {
+    return matcherNames.reduce((acc, matcherName) => {
+        acc[matcherName] = mockMatcher(matcherName);
+        return acc;
+    }, {} as Record<string, MockMatcherFn>);
+}
+
+function mockMatcher(matcherName: string): MockMatcherFn {
```

Comment:
There's a complicated logic here that I've used from wdio - https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-browser-runner/src/browser/expect.ts#L33-L142